.drone.yml

```diff
@@ -43,7 +40,47 @@ steps:
     depends_on:
     - frontend
     - backend
-  - name: release
+
+---
+kind: pipeline
+name: nightly
+
+workspace:
+  base: /go
+  path: src/github.com/vmware/octant
+
+steps:
+  - name: fetch
+    image: docker:git
+    commands:
+      - git fetch --tags
+  - name: releaser
+    environment:
+      GOOGLE_APPLICATION_JSON:
+        from_secret: google_application_json
+    trigger:
+      event:
+      - cron
+      cron:
+      - nightly
+    image: goreleaser/goreleaser:v0.119-cgo
+    commands:
+      - echo $GOOGLE_APPLICATION_JSON > gs.json
+      - GOOGLE_APPLICATION_CREDENTIALS=gs.json goreleaser -f .goreleaser-nightly.yml --rm-dist --skip-validate
```

Comment:
I can't see the credentials in drone, but there would be two things to check:
- Are the credentials being retrieved correctly?
- Should you use a full path for `GOOGLE_APPLICATION_CREDENTIALS`?

Reply:
I moved everything in to a bash script to give me better control over it. So I'll see if that helps after this next run.